pkg/blockdevice/v1alpha1/blockdevice.go

```diff
@@ -211,6 +212,34 @@ func (bdl *BlockDeviceList) Filter(predicateKeys ...string) *BlockDeviceList {
 	return filteredBlockDeviceList
 }
 
+// GetUsableBlockDevices returns custom blockdevice list which can be used
```

Comment:
What is a custom block device list ?

Reply:
Custom block device list is wrapper over core api object list

Reply:
Done